docs/modules/ROOT/pages/ios_files.adoc

```diff
@@ -0,0 +1,175 @@
+image:file-actions-multiple-files-selected.png[File actions dialog for multiple-selected files in ownCloud's iOS App.]
+|===
+
+== Navigate Folders
+
+There are two ways to navigate folders (outside of the root folder).
+To go back to the parent folder, tap btn:[Back] in the top left-hand corner.
+To navigate directly to *any* parent folder tap on the current folder's name.
+When you do, you will see the names of all the parent folders, right up to the root folder, as in the image below.
+
+image:21_File_list_parent.png[Navigating folders in ownCloud's iOS app.]
+
+== View Files
+
+To view a file, tap on its name in the file list.
+Any file type {ios-safari-supported-filetypes-url}[supported by iOS Safari] can be displayed in the app.
+Depending on the file type, the image will be able to viewed, or an icon for it, along with some file details, will be displayed.
+
+NOTE: If the file is not available locally on the device, you will see image:icon-not-available-locally.png[alt=A file is not downloaded locally on the ownCloud iOS app,width=25] next to the file.
+When you click on it, you will see image:icon-download.png[alt=A file is downloading on the ownCloud iOS app,width=20] next to it while it downloads.
+
+[cols=",,"]
+.Viewing different file types
+|===
+a|
+.An image file
+image:view-file-image.png[Viewing an image file in the ownCloud iOS App]
+a|
+.A video file
+image:view-file-video.png[Viewing a video file in the ownCloud iOS App]
+a|
+.A PDF file
+image:view-file-pdf.png[Viewing a PDF file in the ownCloud iOS App]
+a|
+.A text file
+image:view-file-text-file.png[Viewing a text file in the ownCloud iOS App]
+a|
+.An ODT file.
+image:view-file-odt.png[Viewing an ODT file in the ownCloud iOS App]
+|
+|===
+
+=== PDF Files
+
+When viewing PDF files four UI options are available which make working with them easier; these are:
+
+* A page selector
+* Page thumbnails
+* A Table of Contents
+* File Search
+
+You can see an example of each in the images below.
+
+[cols=",,,"]
+.PDF file functionality
+|===
+a|
+.A page selector
+image:41_PDF.png[Page selector in PDF files in the ownCloud iOS App]
+a|
+.A table of contents
+image:42_PDF_toc.png[Table of contents in PDF files in the ownCloud iOS App]
+a|
+.Page thumbnails
+image:43_PDF_thumbs.png[Page thumbnails in PDF files in the ownCloud iOS App]
+a|
+.File search
+image:44_PDF_search.png[File search in PDF files in the ownCloud iOS App]
+a|
+|===
+
+=== Video Files
+
+Video files have the standard iOS video controls available, which include play, pause, AirPlay, volume, skip forward, skip back, close, and full screen.
+
+== Folder Actions
+
+When working with folders, click the plus icon near the top right-hand corner, and three actions become available; these are:
+
+* xref:create-folder[Create folder]
+* xref:upload-files[Upload files]
+* xref:upload-file-from-your-photo-library[Upload file from your photo library]
+
+image:directory-actions.png[Folder actions in ownCloud's iOS App.]
```

Comment:
Screenshot outdated

Reply:
Won't fix for now – screenshots need to be updated anyway